discord/ext/commands/help.py

```diff
@@ -1340,3 +1341,168 @@ async def send_command_help(self, command):
         self.add_command_formatting(command)
         self.paginator.close_page()
         await self.send_pages()
+
+        
+class EmbeddedHelpCommand(HelpCommand):
+    """The implementation of the embedded help command.
+    
+    This inherits from :class:`HelpCommand`
+    
+    It extends it by formatting prefixed command help data into a more attractive embedded form.
+    """
+
+    
+    async def send_bot_help(self, mapping):
+        ctx = self.context
+        bot = self.context.bot
+        
+        # Sort commands by their parent cog
+        cog_command_mapping = {}
+        for cmd in bot.commands:
+            if cmd.hidden == True:
+                continue
+            if cmd.cog:
+                cog_name = cmd.cog.qualified_name if cmd.cog.qualified_name is not None else "Uncategorized Commands"
```

Comment:
This isn't right because technically these commands are "categorized" under a cog, it just has no qualified name